dnf-behave-tests/features/plugins-core/builddep-modularity.feature

```diff
@@ -0,0 +1,90 @@
+Feature: Tests for builddep command on modular system
+
+Background: Enable builddep plugin
+  Given I enable plugin "builddep"
+    And I use the repository "dnf-ci-fedora"
+
+# module ninja:master [d] contains ninja-build-0:1.8.2-4.module_1991+4e5efe2f.x86_64
+# ninja:legacy contains ninja-build-0:1.5.2-1.module_1991+4e5efe2f.x86_64
+# ninja:development contans ninja-build-1.9.2-1.module_1991+4e5efe2f.x86_64
+# ursine dnf-ci-fedora repo contains ninja-build-0:1.8.2-5.fc29.x86_64
+
+Scenario: Builddep installs non-modular build requirements
+   When I execute dnf with args "builddep {context.dnf.fixturesdir}/repos/dnf-ci-builddep/src/build-requires-ninja-build-1.0-1.src.rpm"
+   Then the exit code is 0
+    And Transaction is following
+        | Action        | Package                               |
+        | install       | ninja-build-0:1.8.2-5.fc29.x86_64     |
+
+
+# do not run on Fedora until issue fixed in libsolv
+# https://github.com/rpm-software-management/libsolv/pull/1
+@use.with_os=rhel__ge__8
+@1677583
+Scenario: Builddep preferes default stream over other streams / non-modular content even though the version is older
+  Given I use the repository "dnf-ci-fedora-modular"
+   When I execute dnf with args "builddep {context.dnf.fixturesdir}/repos/dnf-ci-builddep/src/build-requires-ninja-build-1.0-1.src.rpm"
+   Then the exit code is 0
+    And Transaction contains
+        | Action                | Package                                           |
+        | install               | ninja-build-0:1.8.2-4.module_1991+4e5efe2f.x86_64 |
+        | module-stream-enable  | ninja:master                                      |
+
+
+@use.with_os=rhel__ge__8
+@1677583
+Scenario: Builddep preferes enabled stream over other streams / non-modular content even though the version is older
+  Given I use the repository "dnf-ci-fedora-modular"
+   When I execute dnf with args "module enable ninja:legacy"
+   Then the exit code is 0
+   When I execute dnf with args "builddep {context.dnf.fixturesdir}/repos/dnf-ci-builddep/src/build-requires-ninja-build-1.0-1.src.rpm"
+   Then the exit code is 0
+    And Transaction contains
+        | Action                | Package                                           |
+        | install               | ninja-build-0:1.5.2-1.module_1991+4e5efe2f.x86_64 |
+
+
+# build-requires-ninja-build-0.5-1.src.rpm build requires ninja-build < 1.8
+@1677583
+Scenario: Builddep reports error where required package is available in non-enabled non-default stream
```

Comment:
I am missing there a word "only" here - only in non-enabled non-default stream. Because if it's also in some enabled stream it doesn't report the error.
Otherwise, LGTM.

Reply:
Fixed.